docs/development/build-instructions-windows.md

```diff
@@ -1,29 +1,30 @@
-# Build instructions (Windows)
+# Build Instructions (Windows)
+
+Follow the guidelines below for building Electron on Windows.
 
 ## Prerequisites
 
 * Windows 7 / Server 2008 R2 or higher
 * Visual Studio 2013 - [download VS 2013 Community Edition for
-  free](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+  free](https://www.visualstudio.com/downloads/download-visual-studio-vs).
 * [Python 2.7](http://www.python.org/download/releases/2.7/)
 * [Node.js](http://nodejs.org/download/)
-* [git](http://git-scm.com)
+* [Git](http://git-scm.com)
 
-If you don't have a Windows installation at the moment,
-[modern.ie](https://www.modern.ie/en-us/virtualization-tools#downloads) has
-timebombed versions of Windows that you can use to build Electron.
+If you don't currently have a Windows installation, [modern.ie](https://www.modern.ie/en-us/virtualization-tools#downloads)
+has timebombed versions of Windows that you can use to build Electron.
 
-The building of Electron is done entirely with command-line scripts, so you
-can use any editor you like to develop Electron, but it also means you can
-not use Visual Studio for the development. Support of building with Visual
-Studio will come in the future.
+Building Electron is done entirely with command-line scripts and cannot be done
+with Visual Studio. You can develop Electron with any editor but support for
+building with Visual Studio will come in the future.
 
 **Note:** Even though Visual Studio is not used for building, it's still
 **required** because we need the build toolchains it provides.
 
-**Note:** Visual Studio 2015 will not work. Please make sure to get MSVS **2013**.
+**Note:** Visual Studio 2015 will not work. Please make sure to get MSVS
+**2013**.
 
-## Getting the code
+## Getting the Code
 
 ```powershell
 git clone https://github.com/atom/electron.git
```

Comment:
In the other build docs we use the `$ <command>` syntax but we don't here—I'm not sure if that's just not commonly used in Windows docs or if it was just left out and is fine to add for consistency? @zcbenz